.github/workflows/build-and-test.yml

```diff
@@ -59,3 +59,13 @@ jobs:
       - name: CMake Build
         shell: pwsh
         run: cmake --build .\build
+
+  go-no-go: # Required status check
+    name: Go/No-Go
```

Comment:
```suggestion
    name: Go / No-Go
```

Reply:
Good call... And here I was saying that the name would be stable

Reply:
And that slash might be confusing with the workflow / job separator. Maybe it needs a different name?

Reply:
`validated`?

Reply:
The display name is what is important. I changed it to `Merge Policy`. That should be clear and stable over time.